_data/sites.json

```diff
@@ -9663,10 +9663,10 @@
     },
     {
         "name": "Indeed",
-        "url": "https://indeed.com",
-        "difficulty": "easy",
-        "notes": "You must login, go to your account settings and then click on the 'Close my account'.",
-        "notes_tr": "Giriş yapın, \"Hesap Ayarları\" yerine girin ve \"Hesabımı Kapat\" tuşuna tıklayın.",
+        "url": "https://requests.hrtechprivacy.com/create?entity=INDEED",
+        "difficulty": "hard",
+        "notes": "To close your account: go to https://secure.indeed.com/settings/account and then click on 'Close my account'. To DELETE your data: go to linked URL and fill out the ticket.",
```

Comment:
Our service is about deleting accounts, so I think we should swap it here: have the URL to delete account above and the URL to delete data in the notes. Also you can use markdown to make prettier hyperlinks on the notes!